draft-west-cookie-incrementalism.md

```diff
@@ -401,85 +404,361 @@ from "have the same registrable domain" (or "is an exact match for") to say
 
 Note: The request's URL when establishing a WebSockets connection has scheme "http" or "https", rather than "ws" or "wss". FETCH maps schemes when constructing the request. This mapping allows same-site cookies to be sent with WebSockets.
 
-## Scheming Cookies {#scheming-cookies}
+## Origin-Bound Cookies {#origin-bound-cookies}
+
+Cookies are one of the few components of the web platform that are not
+scoped to the origin by default. This difference in scoping means that
+cookies have weaken confidentiality and integrity compared with other
+storage APIs on the web platform.
+
+Examples:
+
+1. `https://somesite.com` sets a simple cookie, `secret=123456`,
+    which contains private information about a user. Information that
+    an attacker wishes to learn. To do so the attacker
+    man-in-the-middles the user, and then tricks them into visiting
+    `http://somesite.com` (note the insecure scheme). When the user
+    visits that page their browser will send the `secret` cookie and
+    the attacker can see it.
+
+2. Similarly, if the attacker has somehow compromised a service running
+    on a different port on the same server, let's say port 345, as
+    `https://somesite.com` then they could trick the user into visiting
+    `https://somesite.com:345`, the user's browser will send the `secret`
+    cookie, and once again the attacker can see it.
+
+Even more, through the same techniques, an attacker can also modify
+a user's cookies, sending a `Set-Cookie` field instead of simply
+eavesdropping.
+
+All of these examples are possible because cookies by default do not
+care about the scheme or port of their connection. As long as the host
+matches the cookie will be accessible.
+
+Some of these shortcomings can be alleviated: the `Secure` attribute
+scopes a cookie to only be accesible on secure schemes and the cookie
+prefixes, `__Secure-` and `__Host-`, can tighten that boundary more.
```

Comment:
Pedantic Nit: The prefixes don't tighten the boundary, they prevent the server from accidentally loosening it. The lack of a `Domain` attribute is what makes `__Host-` cookies more tightly scoped.

Reply:
True, I've reworded them to keep the focus on the "secure" attribute.